tests/settings.py

```diff
@@ -97,3 +97,7 @@ def load():
 
 
 MSP_IDS = [n.msp_id for n in _SETTINGS.nodes]
+
+# <Additional network features>
+ENABLE_INTERMEDIATE_MODEL_REMOVAL = os.getenv('ENABLE_INTERMEDIATE_MODEL_REMOVAL') == '1'
+# </End additional network features>
```

Comment:
@samlesu What do you think if this pattern? I went for the simplest solution here. I'm open to your suggestions/guidance :)